test/near_provider.test.js

```diff
@@ -359,6 +360,48 @@ describe('\n---- PROVIDER ----', () => {
             }));
         });
 
+        describe('retrieveNear | eth_retrieveNear', () => {
+          let account, value
+
+          beforeEach(withWeb3(async (web) => {
+            account = utils.nearAccountToEvmAddress(ACCOUNT_ID)
+            value = 6 * (10 ** 18)
+            await web.eth.sendTransaction({
+                from: account,
+                to: account,
+                value,
+                gas: 0
+            });
+          }));
+
+          test('sends near back to nearAccount if sufficient funds in corresponding evm account', withWeb3(async (web) => {
+              let valueRetrieved = value / 2;
+              let prevBalance = parseInt(await web.eth.getBalance(account, 'latest'))
+              let retrieveNear = await web.eth.retrieveNear({
+                from: account,
+                value: valueRetrieved,
+                to: ACCOUNT_ID,
+                gas: 0
+              })
+              let newBalance = parseInt(await web.eth.getBalance(account, 'latest'))
+              expect(prevBalance - newBalance).toStrictEqual(valueRetrieved)
```

Comment:
are there docs on checking near account balances?

Reply:
It should work if gas price is 0

Reply:
Here I'm checking the balance of the evm address inside our near-evm contract. I'd like to check the actual near balance as well. I'm guessing it's possible via the `near-api-js` I'm just having trouble finding docs

Reply:
https://github.com/near/near-api-js/blob/a9fe1282df33a822500a9ac501a9944a6e59f195/src/account.ts#L373